clair/api.go

```diff
@@ -57,7 +58,7 @@ func (a *apiV1) pushLayer(layer *layer) error {
 	}
 	request.Header.Set("Content-Type", "application/json")
 	utils.DumpRequest(request)
-	response, err := a.client.Do(request)
+	response, err := a.sendWithRetries(request)
```

Comment:
there is also `a.client.Do(request)` in the `Analyze` func - we don't need it also there?